src/ui/Templates/TemplateConditionEvaluator.ts

```diff
@@ -11,7 +11,11 @@ export class TemplateConditionEvaluator {
     const fieldBetweenDoubleQuotes = `"[^"]*?(${acceptableCharacterInFieldName}+)[^"]*?"`;
     const fieldBetweenSingleQuotes = `'[^']*?(${acceptableCharacterInFieldName}+)[^']*?'`;
     const rawFieldAccessedUsingString = `\\braw\\[(?:${fieldBetweenDoubleQuotes}|${fieldBetweenSingleQuotes})\\]`;
-    const fieldMatcher = new RegExp(`${fieldWithAtSymbolPrefix}|${rawFieldAccessedUsingDotOperator}|${rawFieldAccessedUsingString}`, 'gi');
+    const fieldUsedInCondition = `data-condition-field-[not-]{0,4}(${acceptableCharacterInFieldName}+)=`;
```

Comment:
`[not-]{0-4}` will also match incomplete names, I think this is the regex you're looking for:
```ts
const fieldUsedInCondition = `data-condition-field-(?:not-)?(${acceptableCharacterInFieldName}+)=`;
```

Also, should we add new tests in `TemplateConditionEvaluatorTest` for this case?

Reply:
Bon Point, j'en ajoute.